src/managers/BreakpointManager.ts

```diff
@@ -166,8 +168,8 @@ export class BreakpointManager {
                     breakpoint.line,
                     breakpoint.column,
                     [
-                        project.rootDir,
-                        ...project.sourceDirs
+                        ...project.sourceDirs,
+                        project.rootDir
```

Comment:
Interesting. I didn't think this was right at first, but now that I think about it more, I'm starting to see the issue. 

In situations when we are using sourceDirs, the `rootDir` becomes kind of a "staging folder" itself. In that situation, we actually want to walk backwards through the sourceDirs first. Otherwise, we are most likely going to find the file in `rootDir` every time. So by moving it to the end, that solve the issue. Nice catch.

Reply:
Exactly.